stack.yaml

```diff
@@ -13,6 +13,8 @@ packages:
 - lib/text-class
 - lib/test-utils
 - lib/shelley
+- ../cardano-addresses/command-line
```

Comment:
this is interim. when cardano-addresses 3.2.0 lands it should be removed. I am doing this for the sake of local testing